owtf/managers/config.py

```diff
@@ -5,73 +5,96 @@
 """
 import logging
 import os
+import yaml
 
 from owtf.config import config_handler
 from owtf.db import models
 from owtf.lib.exceptions import InvalidConfigurationReference
 from owtf.utils.error import abort_framework
 from owtf.utils.file import FileOperations
 from owtf.utils.strings import multi_replace, str2bool
+from owtf.utils.pycompat import get_dict_iter_items
 
-try:
-    import configparser as parser
-except ImportError:
-    import ConfigParser as parser
 
+def load_config_file(file_path, fallback_file_path):
+    """Load YAML format configuration file
+
+    :param file_path: The path to config file
+    :type file_path: `str`
+    :param fallback_file_path: The fallback path to config file
+    :type fallback_file_path: `str`
+    :return: config_map
+    :rtype: dict
+    """
+    file_path = file_path if not os.path.isfile(file_path) else fallback_file_path
```

Comment:
This logic is incorrect. It should be `file_path if os.path.isfile(file_path) else fallback_file_path` since we want to load files from `~/.owtf` over `owtf/data`.